### PR TITLE
Proper RPM packaging to take into account the changes from #2

### PR DESCRIPTION
**Pull request by Rudd-O**
Here you go.  Thank you so much for the review.

### CODE REVIEW

File: seedvault-backup-parser.spec [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           seedvault-backup-parser
+Version:        2020.10.23
+Release:        %{mybuildnumber}%{?dist}
+Summary:        Parse Seedvault backups
+BuildArch:      noarch
+
+License:        GPLv2+
```
Comment (tlambertz) [inline seedvault-backup-parser.spec]: I have licensed the script under Apache 2.0, not GPL.

**Review by tlambertz — CHANGES_REQUESTED** [review]
I don't have an RPM based system available for testing, but your pull looks sane. I'll merge if you fix the license.

File: seedvault-backup-parser.spec (hunk 1 above)
Comment (Rudd-O) [reply seedvault-backup-parser.spec]: It's now resolved.

**Comment by tlambertz** [comment]
Thank you!

**Pull request merged**